sdk/core/Microsoft.Extensions.Azure/README.md

```diff
@@ -0,0 +1,75 @@
+# Azure client integration for ASP.NET Core
+
+Microsoft.Extensions.Azure.Core provides shared primitives to integrate Azure clients with ASP.NET Core [dependency injection][dependency_injection] and [configuration][configuration] systems.
+
+[Source code][source_root] | [Package (NuGet)][package]
+
+## Installing
+
+Install the ASP.NET Core integration library using [NuGet][nuget]:
+
+```
+dotnet add package Microsoft.Extensions.Azure
+```
+
+## Usage Scenarios and Samples
+
+Make a call to `AddAzureClients` in your app's `ConfigureServices` method. You can use the provided builder to register client instances with your dependency injection container.
+
+```C# Snippet:ConfigureServices
+public void ConfigureServices(IServiceCollection services)
+{
+    // Registering policy to use in ConfigureDefaults later
+    services.AddSingleton<DependencyInjectionEnabledPolicy>();
+
+    services.AddAzureClients(builder => {
+        // Register blob service client and initialize it using the KeyVault section of configuration
+        builder.AddSecretClient(Configuration.GetSection("KeyVault"))
```

Comment:
Would it make sense to have an example of the .config file that corresponds to this?

Reply:
Yes, adding.